Dockerfile

```diff
@@ -0,0 +1,80 @@
+ARG DISTRO=debian
+ARG VERSION=buster
+FROM $DISTRO:$VERSION
+
+RUN export DEBIAN_FRONTEND=noninteractive && \
+    apt-get update && \
+    apt-get install -y \
+    build-essential \
+    crossbuild-essential-armhf  \
+    dh-make \
+    git \
+    wget \
+    curl \
+    debhelper \
+    devscripts \
+    equivs
+
+ARG ARCH=amd64
+ARG DISTRO=debian
+ARG VERSION=buster
+
+ENV ARMV6_TOOLCHAIN_ROOT="/opt/armv6-eabihf--glibc--stable-2020.08-1"
+
+RUN if [ "${DISTRO}" = "ubuntu" -a "${ARCH}" != "amd64" ]; then \
+        sed -i 's/^deb/deb [arch=amd64]/' /etc/apt/sources.list && \
+        touch /etc/apt/sources.list.d/ports.list && \
+        echo "deb [arch=$ARCH] http://ports.ubuntu.com/ubuntu-ports/ $VERSION main restricted universe multiverse" > /etc/apt/sources.list.d/ports.list && \
+        echo "deb [arch=$ARCH] http://ports.ubuntu.com/ubuntu-ports/ $VERSION-updates main restricted universe multiverse" >> /etc/apt/sources.list.d/ports.list && \
+        echo "deb [arch=$ARCH] http://ports.ubuntu.com/ubuntu-ports/ $VERSION-backports main restricted universe multiverse" >> /etc/apt/sources.list.d/ports.list && \
+        echo "deb [arch=$ARCH] http://ports.ubuntu.com/ubuntu-ports/ $VERSION-security main restricted universe multiverse" >> /etc/apt/sources.list.d/ports.list; \
+    fi
+
+RUN dpkg --add-architecture ${ARCH} && \
+    apt-get update
+
+RUN if ! [ "${DISTRO}" = "debian" -a "${ARCH}" = "armhf" ]; then \
+        apt-get install -y \
+        pkg-config \
+        liblzma-dev:${ARCH} \
+        libssl-dev:${ARCH} \
+        libglib2.0-dev:${ARCH} \
+        libmount-dev:${ARCH} \
+        libc-dev:${ARCH} \
+        libc6-dev:${ARCH} \
+        linux-libc-dev:${ARCH} \
+        ; \
+    fi
+
+
+COPY install-armv6-cross-toolchain /bin
+RUN chmod +x /bin/install-armv6-cross-toolchain
+
+RUN if [ "${DISTRO}" = "debian" -a "${ARCH}" = "armhf" ]; then \
+      ARMV6_TOOLCHAIN="armv6-eabihf--glibc--stable-2020.08-1" \
+      VERSION=${VERSION} \
+        /bin/install-armv6-cross-toolchain; \
+    fi
+
+RUN if [ "${ARCH}" = "arm64" ]; then \
+        apt-get install -y gcc-aarch64-linux-gnu; \
+    fi
+
+RUN if [ "${ARCH}" = "armhf" -a "${DISTRO}" != "debian" ]; then \
+        apt-get install -y gcc-arm-linux-gnueabihf; \
+    fi
+
+# Import GPG key, if set
+ARG GPG_KEY_BUILD=""
+RUN if [ -n "$GPG_KEY_BUILD" ]; then \
+        echo "$GPG_KEY_BUILD" | gpg --import; \
+    fi
+
+# TODO: Figure out why these packages are required
+RUN if [ "${DISTRO}" = "debian" -a "${ARCH}" = "armhf" ]; then \
+        apt-get install -y libmount1:armhf zlib1g:armhf; \
+    fi
+
+# Prepare the deb-package script
+COPY mender-deb-package /usr/local/bin/
+ENTRYPOINT  ["/usr/local/bin/mender-deb-package"]
```

Comment:
The `Dockerfile`, together with the new script `install-armv6-cross-toolchain`, needs to be submitted to `mender-test-containers` instead ([here](https://github.com/mendersoftware/mender-test-containers/tree/master/mender-dist-packages-building)).